petastorm/reader.py

```diff
@@ -201,11 +204,12 @@ def make_reader(dataset_url,
 def make_batch_reader(dataset_url_or_urls,
                       schema_fields=None,
                       reader_pool_type='thread', workers_count=10,
+                      shuffle_rows=False, shuffle_seed=None,
                       shuffle_row_groups=True, shuffle_row_drop_partitions=1,
                       predicate=None,
                       rowgroup_selector=None,
                       num_epochs=1,
-                      cur_shard=None, shard_count=None, shard_seed=None,
```

Comment:
Can we keep both seeds for now for the next couple of versions and add a deprecation warning? That way we don't break compatibility but give users a warning.